web/angular-wallet/src/app/main/messages/sidenavs/right/options/options.component.html

```diff
@@ -24,7 +24,7 @@
   <div class="mat-form-field">
     <burst-fee-selector
       [fees]="fees"
-      [(feeNQT)]="feeNQT"
+      [(feeNQT)]="feeBurst"
```

Comment:
wtf! changing the selectors attribute name is veeeery risky...so I didn't change